administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -63,35 +198,39 @@ function triggerSave() {
 		height: 1500px;
 		border: 0 !important;
 	}
+
+	.reference-side h3 {
+		float: left;
+		width: 50%;
+	}
 ');
 ?>
-<button id="toogle-left-panel">Show/Hide Reference (PoC)</button>
+<button id="toogle-left-panel" class="btn btn-small">Show/Hide Reference (PoC)</button>
 
-<form action="<?php echo JRoute::_('index.php?option=com_associations&view=association'); ?>"
- method="post" name="adminForm" id="item-form" class="form-validate">
+<form action="<?php echo JRoute::_(JFactory::getURI()->toString()); ?>"
+ method="post" name="adminForm" id="adminForm" class="form-validate">
 
 <div class="sidebyside">
 	<div class="outer-panel" id="left-panel">
 		<div class="inner-panel">
 			<h3><?php echo JText::_('COM_ASSOCIATIONS_REFERENCE_ITEM'); ?></h3>
-			<iframe src="<?php echo JRoute::_($this->link); ?>" 
+			<iframe id="reference-association" src="<?php echo JRoute::_($this->link); ?>" class="reference-association"
 				name="<?php echo JText::_('COM_ASSOCIATIONS_TITLE_MODAL'); ?>" height="100%" width="400px" scrolling="no">
 			</iframe>
 		</div>
 	</div>
 	<div class="outer-panel" id="right-panel">
 		<div class="inner-panel">
-			<h3><?php echo JText::_('COM_ASSOCIATIONS_ASSOCIATED_ITEM'); ?></h3>
-			<select id="ref-language" name="ref-language">
-				<?php echo JHtml::_('select.options', JHtml::_('contentlanguage.existing', false, true), 'value', 'text'); ?>
-			</select>
-			<button onclick="return triggerSave();" class="btn btn-small btn-success">
-				<span class="icon-apply icon-white"></span>Save
-			</button>
+			<div class="reference-side">
```

Comment:
If you can tell me a way to have the filter and the label "Target" in the same line without doing this please tell me :smiley: